ui/dashboards/src/components/Panel/PanelHeader.tsx

```diff
@@ -33,10 +33,20 @@ export interface PanelHeaderProps extends Omit<CardHeaderProps, OmittedProps> {
   };
 }
 
-export function PanelHeader({ id, title, description, editHandlers, sx, ...rest }: PanelHeaderProps) {
+export function PanelHeader({
+  id,
+  title: rawTitle,
+  description: rawDescription,
+  editHandlers,
+  sx,
+  ...rest
+}: PanelHeaderProps) {
   const titleElementId = `${id}-title`;
   const descriptionTooltipId = `${id}-description`;
 
+  const title = useReplaceVariablesInString(rawTitle);
+  const description = useReplaceVariablesInString(rawDescription ?? null);
```

Comment:
Ah, guessing this is where the `null` came from (I was looking at the code out of order). Would an `undefined` make slightly more sense here? I find that's much more commonly used for the "there's nothing here" case in TypeScript (generally) and this codebase (specifically).